test/jdk/java/io/File/libGetXSpace.c

```diff
@@ -125,7 +125,7 @@ Java_GetXSpace_getSpace0
     }
 #else
     struct statfs buf;
-    int result = statfs((const char*)chars, &buf);
+    int result = statfs((char*)chars, &buf);
```

Comment:
Is this working around a bug in IBM's declaration?

Also, pre-existing, the cast seems really suspicious.  The type of `chars` is `jchar*`, which is a
sequence of 16bit characters.  Does this actually work?  If so, how?

Reply:
This is IBMs declaration of statfs
`extern int statfs(char *, struct statfs *);`
So the compiler will not accept a `const char*`
Indeed I do not know if this ever worked, but my change makes it not worse.

Reply:
Here is the documentation of statfs on AIX  https://www.ibm.com/docs/en/aix/7.2?topic=s-statfs-fstatfs-statfs64-fstatfs64-ustat-subroutine  (showing the IBM declaration Joachim told us) .